script/reportbuild/upload-build-results.js

```diff
@@ -0,0 +1,39 @@
+const args = require('minimist')(process.argv.slice(2))
+const assert = require('assert')
+const octokit = require('@octokit/rest')()
+const s3Url = 'https://s3.amazonaws.com/github-janky-artifacts/libchromiumcontent'
+
+assert(process.env.LIBCHROMIUMCONTENT_GITHUB_TOKEN, 'LIBCHROMIUMCONTENT_GITHUB_TOKEN not found in environment')
+
+async function postBuildResults () {
+  octokit.authenticate({
+    type: 'oauth',
+    token: process.env.LIBCHROMIUMCONTENT_GITHUB_TOKEN
+  })
+
+  try {
+    let buildSummary
+    if (args.failed) {
+      buildSummary = `:x: ${args.buildName} failed for ${args.commitId}.`
+    } else {
+      buildSummary = `:white_check_mark: ${args.buildName} succeeded for ${args.commitId}.`
+    }
+    let body = `${buildSummary}  [Details](${s3Url}/${args.logFile})`
```

Comment:
Asking because I don't know how this expansion works, do we need to do any sanitization on this? It _looks_ like it is done for us, but am unsure so asking